service/routes.py

```diff
@@ -63,11 +63,30 @@
     },
 )
 
+create_model = api.model(
```

Comment:
I feel that `shopcart_base_model` may be a better naming than `create_model`; anyway, I can propose corresponding change in another PR 🙂

Reply:
sure!